### PR TITLE
Feat: Text utility component is added

### PR DESCRIPTION
**Pull request by sakleprajwal**
Text utility component is added in the component library.
Preview:

image

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -6,6 +6,7 @@
 @import "./component\ styles/Image.css";
 @import "./component\ styles/Input.css";
 @import "./component\ styles/Navigation.css";
+@import "./component\ styles/Text-utility.css";
```
Comment (thesudhanshupanda) [inline style.css:9]: you can import by using the below approach , 

example:
@import url("./images.css");

**Review by thesudhanshupanda — APPROVED** [review]
good to merge , just check the comment once .

File: component styles/Text-utility.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    margin: 0 auto;
+    max-width: 40rem; 
+}
+
+.text-utility-container h1,h3{
+    margin-top: 2rem;
+}
+
+.text-utility-container p{
+    margin: 1rem 0 1rem 0;
```
Comment (kotalwar-krishna) [inline component styles/Text-utility.css:12]: margin: 1rem 0; like short form you can use

**Review by kotalwar-krishna — APPROVED** [review]
Good to merge

**Pull request merged**